### PR TITLE
Out of support

### PR DESCRIPTION
**Pull request by Rick-Anderson**
---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [aspnetcore/introduction-to-aspnet-core.md](https://github.com/dotnet/AspNetCore.Docs/blob/9fd69238643cb8a6927f0837177cba92c4eafb44/aspnetcore/introduction-to-aspnet-core.md) | [Overview of ASP.NET Core](https://review.learn.microsoft.com/en-us/aspnet/core/introduction-to-aspnet-core?branch=pr-en-us-32877) |

### CODE REVIEW

**Comment by guardrex** [comment]
Shouldn't Line 1 of `not-latest-version.md` be ...

```markdown
:::moniker range=">= aspnetcore-7.0 < aspnetcore-8.0"
```

... and the original ordering in the file doesn't match our usual convention of earliest to latest for versioning blocks ... but that doesn't really matter all that much. It's just a consistency super NIT.

File: aspnetcore/includes/not-latest-version.md [hunk 1]
```diff
@@ -21,3 +21,4 @@ moniker range section until the new moniker is created.
 Markdown to include this file:
 [!INCLUDE[](~/includes/not-latest-version.md)]
 -->
+
```
Comment (Rick-Anderson) [inline aspnetcore/includes/not-latest-version.md:24]:
```suggestion
:::moniker range="< aspnetcore-6.0"
This version of ASP.NET Core is no longer supported. Use the version selector in the documentation to select a supported version. For more information, see the [.NET and .NET Core Support Policy](https://dotnet.microsoft.com/platform/support/policy/dotnet-core) page.
![image](https://github.com/dotnet/AspNetCore.Docs/assets/3605364/9145fe9d-017e-40fd-b6b9-7bccd1bd2325)
:::moniker-end

```

File: aspnetcore/includes/not-latest-version.md [hunk 2]
```diff
@@ -21,3 +21,8 @@ moniker range section until the new moniker is created.
 Markdown to include this file:
 [!INCLUDE[](~/includes/not-latest-version.md)]
 -->
+:::moniker range="< aspnetcore-6.0"
```
Comment (Rick-Anderson) [inline aspnetcore/includes/not-latest-version.md:24]:
```suggestion

:::moniker range="< aspnetcore-6.0"
```

File: aspnetcore/includes/not-latest-version.md [hunk 3]
```diff
@@ -1,3 +1,10 @@
...
```
Comment (tdykstra) [inline aspnetcore/includes/not-latest-version.md:1]: this covers 5 and earlier, but 7 is out of support also.

File: aspnetcore/includes/not-latest-version.md [hunk 4]
```diff
@@ -1,3 +1,10 @@
...
+
+This version of ASP.NET Core is no longer supported. Use the version selector in the documentation to select a supported version. For more information, see the [.NET and .NET Core Support Policy](https://dotnet.microsoft.com/platform/support/policy/dotnet-core) page.
+![image](https://github.com/dotnet/AspNetCore.Docs/assets/3605364/9145fe9d-017e-40fd-b6b9-7bccd1bd2325)
```
Comment (tdykstra) [inline aspnetcore/includes/not-latest-version.md:4]: Use the newer image syntax to get automatic borders --
```
:::image type="content" source="<folderPath>" alt-text="<alt text>":::
```

File: aspnetcore/includes/not-latest-version.md [hunk 5]
```diff
@@ -1,3 +1,10 @@
+:::moniker range="< aspnetcore-6.0"
+
+This version of ASP.NET Core is no longer supported. Use the version selector in the documentation to select a supported version. For more information, see the [.NET and .NET Core Support Policy](https://dotnet.microsoft.com/platform/support/policy/dotnet-core) page.
```
Comment (wadepickett) [inline aspnetcore/includes/not-latest-version.md:3]: We are leaving the customer to guess what the minimal supported version is.  Consider being explicit up front.

**Review by wadepickett — APPROVED** [review]
There is an issue with the build to fix yet.  I also suggested we consider being explicit about what version + is supported rather than leaving the customer to guess or have to go find out.  Other than that, looks good.

File: aspnetcore/includes/not-latest-version.md (hunk 5 above)
Comment (Rick-Anderson) [reply aspnetcore/includes/not-latest-version.md:3]: That needs to be changed one a year. If they're looking at an old version, they can look at the support page.

File: aspnetcore/includes/out-of-support.md [hunk 6]
```diff
@@ -0,0 +1,4 @@
+This version of ASP.NET Core is no longer supported. Use the version selector in the documentation to select a supported version. For more information, see the [.NET and .NET Core Support Policy](https://dotnet.microsoft.com/platform/support/policy/dotnet-core) page.
+:::image type="content" source="https://github.com/dotnet/AspNetCore.Docs/assets/3605364/9145fe9d-017e-40fd-b6b9-7bccd1bd2325" alt-text="version selector":::
+
+![image](https://github.com/dotnet/AspNetCore.Docs/assets/3605364/9145fe9d-017e-40fd-b6b9-7bccd1bd2325)
```
Comment (Rick-Anderson) [inline aspnetcore/includes/out-of-support.md:4]:
```suggestion
```

**Comment by Rick-Anderson** [comment]
@tdykstra  how do we get rid of the following error?
<html>
<body>

Line 1: [Warning] No intersection between zone and file level monikers. The result of zone level range string '< aspnetcore-6.0' is 'aspnetcore-1.0', 'aspnetcore-1.1', 'aspnetcore-2.0', 'aspnetcore-2.1', 'aspnetcore-2.2'..., while file level monikers is 'aspnetcore-7.0', 'aspnetcore-8.0', 'aspnetcore-9.0'.
--



</body>
</html>

File: aspnetcore/includes/out-of-support.md [hunk 7]
```diff
@@ -0,0 +1,3 @@
+This version of ASP.NET Core is no longer supported. Use the version selector in the documentation to select a supported version. For more information, see the [.NET and .NET Core Support Policy](https://dotnet.microsoft.com/platform/support/policy/dotnet-core) page.
+:::image type="content" source="https://github.com/dotnet/AspNetCore.Docs/assets/3605364/9145fe9d-017e-40fd-b6b9-7bccd1bd2325" alt-text="version selector":::
```
Comment (tdykstra) [inline aspnetcore/includes/out-of-support.md:2]: Suggest we send them to the current version like we do with not-latest and omit the image of the version selector.
```suggestion
This version of ASP.NET Core is no longer supported. For more information, see [.NET and .NET Core Support Policy](https://dotnet.microsoft.com/platform/support/policy/dotnet-core). For the current release, see the [.NET 8 version of this article](?view=aspnetcore-8.0&preserve-view=true).
```

**Review by tdykstra — APPROVED** [review]
Looks good, I left one suggestion.

File: aspnetcore/includes/out-of-support.md [hunk 8]
```diff
@@ -0,0 +1,2 @@
+> [!WARNING] This version of ASP.NET Core is no longer supported. For more information, see [.NET and .NET Core Support Policy](https://dotnet.microsoft.com/platform/support/policy/dotnet-core). For the current release, see the [.NET 8 version of this article](?view=aspnetcore-8.0&preserve-view=true).
```
Comment (Rick-Anderson) [inline aspnetcore/includes/out-of-support.md:1]:
```suggestion
> [!WARNING]
> This version of ASP.NET Core is no longer supported. For more information, see [.NET and .NET Core Support Policy](https://dotnet.microsoft.com/platform/support/policy/dotnet-core). For the current release, see the [.NET 8 version of this article](?view=aspnetcore-8.0&preserve-view=true).
```

**Comment by Rick-Anderson** [comment]
@tdykstra 
Before:
image

I don't like Bling, but I think this is needed:
image

**Pull request merged**